spec.bs

```diff
@@ -358,19 +666,67 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
 <div algorithm>
   The <dfn method for=Fence>reportEvent(<var ignore>event</var>)</dfn> method steps are:
 
-  1. <span class=XXX>Fill this out!</span>
+  1. Let *instance* be this [=traversable navigable=]'s [=fenced frame config instance=].
+
+  1. If *instance* is null, then return.
+
+  1. If *instance*'s [=fenced frame config instance/mapped url=] is null, then the behavior is
+     unspecified.
+
+  1. If this [=document=]'s [=origin=] and *instance*'s
```

Comment:
Here and elsewhere let's follow the pattern like https://wicg.github.io/turtledove/#:~:text=durationSeconds)%20method%20steps%20are%3A-,If%20this%27s%20relevant%20global%20object%27s%20associated%20Document,-is%20not%20allowed%20to%20use when it comes to accessing the current document.

Reply:
Changed to the [=relevant settings object=] to match FLEDGE spec, but lmk if I'm misunderstanding what that means.